generators/generator-base.js

```diff
@@ -808,7 +808,28 @@ module.exports = class extends PrivateBase {
      * Generate a date to be used by Liquibase changelogs.
      */
     dateFormatForLiquibase() {
-        const now = new Date();
+        let now = new Date();
+        // Run reproducible timestamp when regenerating the project with with-entities option.
+        if (this.options && this.configOptions && this.options.withEntities) {
```

Comment:
that's one lengthy if there, it would be better to refactor it

for instance, l. 816: what do we need to initialize the now variable? I don't think we need options, configOptions & withEntities, wouldn't configOptions be enough?

Reply:
Reproducibly build only makes sense with `jhipster --with-entities` command and it is not set to configOptions.

this.options and this.configOptions test probably com be removed.